agent/agentexec/cli_linux.go

```diff
@@ -56,28 +65,45 @@ func CLI() error {
 		}
 	}
 
-	if *oom == unset {
-		// If an explicit oom score isn't set, we use the default.
-		*oom, err = defaultOOMScore()
-		if err != nil {
-			return xerrors.Errorf("get default oom score: %w", err)
-		}
+	// We drop effective caps prior to setting dumpable so that we limit the
+	// impact of someone attempting to hijack the process (i.e. with a debugger)
+	// to take advantage of the capabilities of the agent process. We encourage
+	// users to set cap_net_admin on the agent binary for improved networking
```

Comment:
Do we mention this in coder.com/docs?

Reply:
honestly not sure

Reply:
Yeah once this is merged in there are no bugs, I'm going to write up some documentation for resource management in Coder